src/common-components/RedirectLogistration.jsx

```diff
@@ -26,6 +26,7 @@ function RedirectLogistration(props) {
 
     // Redirect to Progressive Profiling after successful registration
     if (redirectToWelcomePage) {
+      // TODO: Do we still need this cookie?
```

Comment:
were you able to get its answer?

Reply:
No, I have added it for future reference.